src/partials/servicesEdit.html

```diff
@@ -23,6 +23,14 @@ <h2 ng-if="service._id" translate>Edit Service</h2>
           <input type="checkbox" ng-model="service.hidden" name="hidden" /> <span translate>Check this to hide this service</span>
         </div>
         <div class="form-group">
+          <label class="control-label" for="auto_add" translate>Automatic addition</label>
+          <input type="checkbox" ng-model="service.auto_add" name="auto_add" /> <span translate>Check this to automatically subscribe to this service a user who is checking into one of the operations selected below</span>
```

Comment:
Change the text to: (Yaelle says)
Check this box to automatically subscribe users to your service once they check in to one of the countries below.